Makefile

```diff
@@ -245,8 +245,12 @@ test-local:
 .PHONY: test-e2e
 test-e2e: ## Runs all Thanos e2e docker-based e2e tests from test/e2e. Required access to docker daemon.
 test-e2e: docker
+	@echo ">> cleaning docker environment."
+	@docker system prune -f --volumes
+	@echo ">> cleaning e2e test garbage."
+	@rm -rf ./test/e2e/e2e_integration_test*
 	@echo ">> running /test/e2e tests."
-	@go test -v ./test/e2e/...
+	@go test -failfast -parallel 1 -timeout 5m -v ./test/e2e/...
```

Comment:
Well... for local use, I get that we want to make sure CI and dev is similar, but slowing down dev docker on purpose, so I cannot use the power of my 12CPUs laptop sounds silly ;p Can we make have this as `E2E_TEST_OPTS`? or special `test-e2e-ci` ? (: